platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -4196,6 +4196,9 @@
 
             var values = $scope.currentRecord.fields;
             angular.forEach (values, function(value, key) {
+                if(value.type.defaultName === "str" && value.value === ""){
```

Comment:
![defaultname](https://cloud.githubusercontent.com/assets/20775311/18209691/284976e8-7135-11e6-805c-1f20c1b02c36.png)
Data looks like this. If we want to check the type of field if it is string type i think we need this defaultName value.